docs/manual/job-plugins.md

```diff
@@ -49,6 +49,16 @@ Notification plugins allow Rundeck to communicate changes in job execution state
 - [Email](/manual/notifications/email.md)
 - [Webhooks](/manual/notifications/webhooks.md)
 
+:::warning
```

Comment:
Not convinced these are necessary here.  Maybe just documenting it on the new page you created below that they can revert to the legacy interface but may run into issues.  This new UI was developed due to a scaling bug and is the long term path forward.

Reply:
Updated! @fdevans